bin/test-cleanup

```diff
@@ -2,9 +2,12 @@
 
 set -eu
 
-k8s_context=${1:-""}
 bindir=$( cd "${BASH_SOURCE[0]%/*}" && pwd )
 
+# shellcheck source=_test-helpers.sh
+. "$bindir"/_test-helpers.sh
```

Comment:
Why adding this dependency, and instead just declare `cleanup_usage` and `handle_cleanup_input` here?

Reply:
Pretty much because we do the same thing in `tests`. Just makes those files a little more readable.

Reply:
@alpeb Another very small reason, but we now use `check_linkerd_binary` in this file as well which is imported from `_test-helpers.sh`.